frontend/components/queries/QueriesList/QueriesList.jsx

```diff
@@ -83,7 +83,13 @@ class QueriesList extends Component {
     return (
       <tr>
         <td colSpan={6}>
-          <p>No queries available. Try creating one.</p>
+          <p>
+            No queries available. Try creating one or get started by{" "}
```

Comment:
I think we should use standard HTML where possible, so replacing `{" "}` with `&nbsp;` would be in line with that. This helps all devs continue with the basics in case we need it.